pages/deposit-dates.css

```diff
@@ -1,46 +1,47 @@
-/* TODO: Custom media from layout.css doesn't work */
-@custom-media --screen-until-desktop (max-width: 727px);
-
-.table thead > :first-child {
-  border-top-width: 2px;
+%item {
+  flex: 3 3 90%;
+  margin: 0 var(--component-spacer-x, 1rem) var(--component-spacer-y, 1.5rem);
 }
 
-.row {
+.container {
   display: flex;
-  flex-wrap: nowrap;
-  margin-bottom: 2rem;
-
-  @media (--screen-until-desktop) {
-    flex-wrap: wrap;
+  flex-wrap: wrap;
+  align-content: flex-start;
```

Comment:
This line fixes [the bug](https://github.com/oacore/dashboard/pull/41#issuecomment-568205794).